src/OnePlusBot/Base/CommandHandler.cs

```diff
@@ -32,11 +32,60 @@ public CommandHandler(DiscordSocketClient bot, CommandService commands, IService
         public async Task InstallCommandsAsync()
         {
             _bot.MessageReceived += OnCommandReceived;
-            _bot.MessageReceived += OnMessageReceived;  
+            _bot.MessageReceived += OnMessageReceived;
+            _bot.MessageDeleted += OnMessageRemoved;
+            _bot.MessageUpdated += OnMessageUpdatedAsync;
             _commands.CommandExecuted += OnCommandExecutedAsync;
             await _commands.AddModulesAsync(Assembly.GetEntryAssembly(), _services);
         }
 
+        private async Task OnMessageUpdatedAsync(Cacheable<IMessage, ulong> cacheable, SocketMessage message, ISocketMessageChannel socketChannel)
+        {
+            var channel = socketChannel as SocketTextChannel;
+            var before = await cacheable.GetOrDownloadAsync();
+            var author = before.Author;
+
+            if (
+                before.Author == _bot.CurrentUser || message.Author == _bot.CurrentUser ||
+                before.Content == "" || message.Content == ""
```

Comment:
Incosistent style, we usually have parentheses on the same line as the conditions